css_tasks/task11/js/accordion.js

```diff
@@ -0,0 +1,47 @@
+import {elements} from './elements.js';
+export function accordion() {
+  elements.then((data) => {
+    const {service, image} = data;
+    service.addEventListener('click', (e) => handleAccordion(e.target));
+    const handleAccordion = (item) => {
+      // do nothing if you didn't click on accordion
+
+      if (item.tagName !== 'SUMMARY') return;
+      const parentContainer = item.parentElement.parentElement;
```

Comment:
it is better to avoid things like this as small change to dom can brake everything.